### ISSUE TITLE
User libraries support?

### ISSUE THREAD

**Issue by linux-china** [cbeust/kash#7]
I want to add some libraries for kash, for example, okhttp3, Klock, JSch etc, and I can write my functions with these libraries support.  Cloud I put jars under ~/.kash/lib and kash will load these jars automatically?

**Comment by cbeust** [comment]
Yes, user classpath is on my [TODO](https://github.com/cbeust/kash/blob/master/TODO.md) list as well.

Right now, I'm thinking that user classpath would be in `~/.kash.json` which Kash would read on start up.
